README.md

```diff
@@ -1,2 +1,2 @@
-# HomesFromArchitectsWebPage-frontend [![Build Status](https://travis-ci.org/RandomUserIK/HomesFromArchitectsWebPage-frontend.svg?branch=dev)](https://travis-ci.org/RandomUserIK/HomesFromArchitectsWebPage-frontend) [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=RandomUserIK_HomesFromArchitectsWebPage-frontend&metric=alert_status)](https://sonarcloud.io/dashboard?id=RandomUserIK_HomesFromArchitectsWebPage-frontend) [![Total alerts](https://img.shields.io/lgtm/alerts/g/RandomUserIK/HomesFromArchitectsWebPage-frontend.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/RandomUserIK/HomesFromArchitectsWebPage-frontend/alerts/) [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/RandomUserIK/HomesFromArchitectsWebPage-frontend.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/RandomUserIK/HomesFromArchitectsWebPage-frontend/context:javascript)
+# HomesFromArchitectsWebPage-frontend [![Build Status](https://travis-ci.org/RandomUserIK/HomesFromArchitectsWebPage-frontend.svg?branch=dev)](https://travis-ci.org/RandomUserIK/HomesFromArchitectsWebPage-frontend) [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=RandomUserIK_SpartaWebPage-frontend&metric=alert_status)](https://sonarcloud.io/dashboard?id=RandomUserIK_SpartaWebPage-frontend) [![Total alerts](https://img.shields.io/lgtm/alerts/g/RandomUserIK/HomesFromArchitectsWebPage-frontend.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/RandomUserIK/HomesFromArchitectsWebPage-frontend/alerts/) [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/RandomUserIK/HomesFromArchitectsWebPage-frontend.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/RandomUserIK/HomesFromArchitectsWebPage-frontend/context:javascript)
```

Comment:
Tato by nemalo ostat ako HomesFromArchitectsWebPage ?

Reply:
nie, lebo sa to nerefreshlo na sonare a uz som skusal dat HomesFromArchitectsWebPage len potom nenajde projekt :/